CleanCode.Naming/RectClass.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CleanCode.Naming
+{
+    class RectClass
+    {
+        // First side
+        private int _SideALength;
```

Comment:
Trochę niespójnie: tutaj w nazwie jest podkreślink, a w kolejnej linijce już nie ma, mimo że są to zmienne identycznego typu (private int)